Components/UnorderedList/index.js

```diff
@@ -0,0 +1,18 @@
+import Container from "../Container";
+import styled from "styled-components";
+
+export default function UnorderedList({ $arrayOfBulletPoints }) {
+  const StyledLi = styled.li`
```

Comment:
Since we always try to avoid shorthand notations `StyledListItem` (or something similar) would be a bit more readable

Reply:
We have renamed "StyledLi" as "StyledListItem".